tensorboard/plugins/debugger_v2/debug_data_multiplexer.py

```diff
@@ -446,6 +452,87 @@ def GraphExecutionData(self, run, begin, end, trace_id=None):
             ],
         }
 
+    def GraphOpInfo(self, run, graph_id, op_name):
```

Comment:
Function names should start with lowercase.  I see there's already a local uppercase convention in this file, so OK I guess.

Reply:
The tensorboard code base as a whole has a many examples that violate this style guide, unfortunately:
E.g., https://github.com/tensorflow/tensorboard/search?q=%22def+Add%22&unscoped_q=%22def+Add%22

So I'll leave it as is for now.